components/BackgroundView/BackgroundView.jsx

```diff
@@ -26,7 +26,7 @@ function BackgroundView(props) {
         ...styles.root,
 
         // contain children in safe area
-        paddingTop: insets.top,
+        paddingTop: insets.top || 12,
```

Comment:
not sure if this 12 number is somthing you use in other places, but declaring it as a const somewhere you could reference might be good in that case. 
const DEFAULT_TOP_PADDING = 12;